src/acl/external/kerberos_ldap_group/support_ldap.cc

```diff
@@ -152,7 +152,7 @@ ldap_simple_rebind(LDAP * ld,
 #elif HAVE_LDAP_REBIND_FUNCTION
 #ifndef LDAP_REFERRALS
 #define LDAP_REFERRALS
-#endif
+#endif /* src/acl/external/kerberos_ldap_group/support_ldap.cc */
```

Comment:
Bad copy-paste?

```suggestion
#endif
```

I do not recommend documenting "obvious"/"immediate" `#endifs` but if you insist on documenting this one:

```suggestion
#endif // LDAP_REFERRALS
```